astroquery/desi/tests/test_module.py

```diff
@@ -0,0 +1,132 @@
+import astropy.io.votable
```

Comment:
rename this file from `test_module` to `test_desi`, as well as the remote tests file.